app/grandchallenge/challenges/models.py

```diff
@@ -827,7 +822,7 @@ class ChallengeRequestStatusChoices(models.TextChoices):
         default=0,
         null=True,
         blank=True,
-        help_text="What is your budget for hosting this challenge, if any?",
+        help_text="What is your budget for hosting this challenge? Please be reminded of our <a href='https://grand-challenge.org/challenge-policy-and-pricing/'>challenge pricing policy</a>.",
```

Comment:
The hostname should not be included in the model.

```suggestion
        help_text="What is your budget for hosting this challenge? Please be reminded of our <a href='/challenge-policy-and-pricing/'>challenge pricing policy</a>.",
```